### PR TITLE
Fix import order.

### PR DESCRIPTION
**Pull request by bjlittle**

### CODE REVIEW

**Comment by bjlittle** [comment]
@lbdreyer ping ...

File: lib/iris/tests/unit/fileformats/grib/load_convert/test_grid_definition_template_30.py [hunk 1]
```diff
@@ -31,8 +31,8 @@
 
 import iris.coord_systems
 import iris.coords
-from iris.tests.unit.fileformats.grib.load_convert import empty_metadata
 from iris.fileformats.grib._load_convert import grid_definition_template_30
+from iris.tests.unit.fileformats.grib.load_convert import empty_metadata
```
Comment (lbdreyer) [inline lib/iris/tests/unit/fileformats/grib/load_convert/test_grid_definition_template_30.py:35]: Well spotted!

File: lib/iris/tests/unit/fileformats/grib/load_convert/test_scanning_mode.py [hunk 2]
```diff
@@ -27,7 +27,7 @@
 import iris.tests as tests
 
 from iris.exceptions import TranslationError
-from iris.fileformats.grib._load_convert import scanning_mode, ScanningMode
+from iris.fileformats.grib._load_convert import ScanningMode, scanning_mode
```
Comment (lbdreyer) [inline lib/iris/tests/unit/fileformats/grib/load_convert/test_scanning_mode.py:30]: Do you remember what ordering we used before, if not ascii ordered?

Comment (bjlittle) [reply lib/iris/tests/unit/fileformats/grib/load_convert/test_scanning_mode.py:30]: ascii, from what I recall ... `man ascii` to see this ordering quickly

**Comment by lbdreyer** [comment]
When I was working on bringing iris_grib into iris I made the decision (possibly a bad one) not to modify files if they only thing that was wrong with them was the import order as I wanted the PR to be as small as possible. 
But I do agree that these changes are definitely needed so I will merge this in (after a final check), just beware that does mean [PR #2486](https://github.com/SciTools/iris/pull/2486) will now require much more scrolling so prepare for your second finger to get tired!

File: lib/iris/tests/unit/fileformats/grib/load_convert/test_scanning_mode.py (hunk 2 above)
Comment (lbdreyer) [reply lib/iris/tests/unit/fileformats/grib/load_convert/test_scanning_mode.py:30]: I meant why had someone chosen to put '**s**canning_mode' before '**S**canningMode'. Your correction makes it follow the ascii order of things but there must have been a reason someone did the above (unless the intention was always to use ascii and these are just examples of mistakes)

Comment (bjlittle) [reply lib/iris/tests/unit/fileformats/grib/load_convert/test_scanning_mode.py:30]: I suspect that it just got tacked onto the end, and their focus was not also ordering the symbols that're imported, but ordering the modules

File: lib/iris/tests/unit/fileformats/grib/load_convert/test_vertical_coords.py [hunk 3]
```diff
@@ -30,10 +30,10 @@
 
 from iris.coords import DimCoord
 from iris.exceptions import TranslationError
-from iris.fileformats.grib._load_convert import vertical_coords
 from iris.fileformats.grib._load_convert import \
-    _TYPE_OF_FIXED_SURFACE_MISSING as MISSING_SURFACE, \
-    _MDI as MISSING_LEVEL
+    _MDI as MISSING_LEVEL, \
+    _TYPE_OF_FIXED_SURFACE_MISSING as MISSING_SURFACE
+from iris.fileformats.grib._load_convert import vertical_coords
```
Comment (lbdreyer) [inline lib/iris/tests/unit/fileformats/grib/load_convert/test_vertical_coords.py]: Did you split up these two imports for a reason, could you not do:
```
from iris.fileformats.grib._load_convert import \
    _MDI as MISSING_LEVEL, \
    _TYPE_OF_FIXED_SURFACE_MISSING as MISSING_SURFACE, \
    vertical_coords
```

Comment (bjlittle) [reply lib/iris/tests/unit/fileformats/grib/load_convert/test_vertical_coords.py]: Yup, I could put them all together ... :+1:

**Comment by bjlittle** [comment]
Thanks!

**Pull request merged**